runnable/test34.d

```diff
@@ -707,6 +710,7 @@ void foo35()
         c = 3;
 
         xxx = cast(typeof(xxx))(a + b);
+	version (DMD_InlineAsm)
         asm { int 3; }
```

Comment:
Did you try to use

    version (LDC) ldc.intrinsics.llvm_debugtrap()

(See https://github.com/ldc-developers/druntime/blob/ldc/src/ldc/intrinsics.di#L445)
Otherwise an assert should be used.